docs-www/src/globalReferences.js

```diff
@@ -0,0 +1,18 @@
+import '@brainhubeu/react-file-input/dist/react-file-input.css';
+
+const { default: FileInput } = (() => {
+  if (!global.window) {
+    global.window = {};
+  }
+  // if ('__RC_ENV__' === 'development') {
+    console.log('connecting with local react-carousel source code');
```

Comment:
here should be `react-file-input`

there's an issue https://github.com/brainhubeu/react-carousel/issues/454 to simplify that to running one command so maybe it's worth first to implement it in react-carousel and then copy it to react-file-input

Reply:
Added a new command to run a local version of the library. WDY?

Reply:
we can simplify that later